lisa/sut_orchestrator/azure/features.py

```diff
@@ -956,7 +956,7 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
         # The Ephemeral doesn't support data disk, but it needs a value. And it
         # doesn't need to calculate on intersect
         value.data_disk_iops = 0
-        value.data_disk_size = 0
+        value.data_disk_size = 1
```

Comment:
@squirrelsc We need to be consistent. The disk class below has min value 1 whereas in code above we were using 0. A value of 1 seems more practical which is why I have made the change here. Please let me know if you think otherwise.

```
data_disk_size: search_space.CountSpace = field(
        default_factory=partial(search_space.IntRange, min=1),
        metadata=field_metadata(
            allow_none=True, decoder=search_space.decode_count_space
        ),
    )
```

Reply:
The below logic will assign a data_disk_size in all logic. the "0" is like to declare a variable with default value. 1 and 0 are the same, because it will be overwritten immediately. But 0 is a good default value, when people need to think about why it's 1.

Reply:
Sure. Let's use default of 0 in that case.

Reply:
updated